jose-jwt/jwe/JweJson.cs

```diff
@@ -0,0 +1,49 @@
+﻿#if NETSTANDARD
+namespace Jose.jwe
+{
+    using Newtonsoft.Json;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Internal class used as helper deserialize Json Serialized JWE
+    /// </summary>
+    internal class JweJson

```

Comment:
Helper (internal) class for JSON deserialization. Not actually use for serialization, Could (maybe) be removed if ok to delegate all the (de)serialization to the settings supplied class.